Assets/Scripts/Models/Area/RoomBehavior.cs

```diff
@@ -387,16 +387,15 @@ public RoomBehavior Clone()
         public void Control(Room room)
         {
             this.Room = room;
-            List<Tile> innerTiles = room.GetInnerTiles();
-            List<Tile> borderTiles = room.GetBorderingTiles();
-
-            List<Tile> allTiles = innerTiles.Union(borderTiles).ToList();
+            HashSet<Tile> innerTiles = room.GetInnerTiles();
+            HashSet<Tile> borderTiles = room.GetBoundaryTiles();
+            innerTiles.UnionWith(borderTiles);
```

Comment:
YEah, the main concern was with the name, the removal of the second variable was more keeping the variable space clean than anything... and I suppose it may help very slightly in minimizing the vertical space taken by a function...

And yeah, the multiple lists was unneccessary... I'm assuming at the time I thought I might need them separate for some reason, and never went back and cleaned it up after discovering I didn't.